src/qt/peertablesortproxy.cpp

```diff
@@ -24,6 +24,8 @@ bool PeerTableSortProxy::lessThan(const QModelIndex& left_index, const QModelInd
     switch (static_cast<PeerTableModel::ColumnIndex>(left_index.column())) {
     case PeerTableModel::NetNodeId:
         return left_stats.nodeid < right_stats.nodeid;
+    case PeerTableModel::Age:
+        return left_stats.m_connected > right_stats.m_connected;
```

Comment:
The first commit doesn't build cleanly without the second commit adding this line; they should be squashed into the same commit.

```
qt/peertablesortproxy.cpp:24:13: warning: enumeration value 'Age' not handled in switch [-Wswitch]
    switch (static_cast<PeerTableModel::ColumnIndex>(left_index.column())) {
            ^
1 warning generated.
```

Reply:
I re-reviewed [CONTRIBUTING.md](https://github.com/bitcoin-core/gui/blob/master/CONTRIBUTING.md) - it has been awhile - and will adjust my workflow and PR structure accordingly - thanks for the correction.

Reply:
Great. Yes, each commit should be hygienic (clean debug build and all tests green).